vizro-core/docs/pages/user-guides/extensions.md

```diff
@@ -0,0 +1,105 @@
+# How to extend and customize Vizro dashboards
+
+At its simplest, Vizro enables low-code configuration, but you can also customize it to go beyond its default capabilities:
+
+* **[Vizro customizations](#vizro-customizations)**. You can customize Vizro to extend the default functionality of Vizro and create Python functions to customize charts, tables, dashboard components, callbacks in the form of actions, and reactive HTML components, then plug them directly into the existing Vizro dashboard configuration.
+
+
+* **[Dash customizations](#dash-customizations)**. You can add custom Dash callbacks directly to any Vizro dashboard, enabling you to code beneath the Vizro layer and control Dash directly.
+
+
+* **[React.js customizations](#reactjs-customizations)**. You can add custom React.js components directly to any Vizro dashboard, enabling users to go beneath both the Vizro and Dash layers and control React.js directly
+
+
+Vizro offers the ability to combine ease of use of low-code configuration, with the flexibility of a range of high-code extensions to expand your dashboard's functionality.
+
+
+## Vizro customizations
+
+Vizro custom functions can be used to extend the default functionality of Vizro. You can create Python functions for customizing charts, tables, dashboard components
+callbacks in the form of actions, and reactive HTML components - then plug them directly
+into the existing Vizro dashboard configuration.
+
+### [Custom charts](custom-charts.md)
+
+You can create custom chart functions in Vizro by wrapping Plotly chart code inside a
+Vizro chart function wrapper, and then use the functions directly inside the Vizro dashboard configuration. This enables the creation of `plotly.graph_objects` charts with multiple traces, or `plotly_express` charts with post update customizations
```

Comment:
Likewise, is it obvious how to "use the functions directly" or should we link?

Reply:
In theory we have a link to the relevant docs already, but it's in the title. Would you prefer to repeat the reference again here?

Reply:
> In theory we have a link to the relevant docs already, but it's in the title. Would you prefer to repeat the reference again here?

If it's just that documentation, then no, that's fine, it's there and one link is sufficient. Just wondered if there was anything else needed.